Firebase/InstanceID/FIRInstanceIDTokenManager.h

```diff
@@ -38,6 +38,12 @@ typedef NS_OPTIONS(NSUInteger, FIRInstanceIDInvalidTokenReason) {
 /// Expose the auth service, so it can be used by others
 @property(nonatomic, readonly, strong) FIRInstanceIDAuthService *authService;
 
+/**
+ * A string containing info about SDK components, Xcode version, etc.
+ * It will be added to each token request to track stats.
```

Comment:
Nit: expanding a bit more would be good to clarify the "stats" part. Something like:

`It will be added to each token request to count third party libraries and version numbers.`